crates/basilica-sdk-python/tests/test_distributed.py

```
@@ -1238,3 +1240,368 @@ def test_distributed_training_facade_reads_via_wrapper(self) -> None:
+
+
+# =============================================================================
+# Phase 5b (#445) -- distributed UD completion semantics.
+# =============================================================================
+
+
+def _make_terminal_training(
+    name: str = "dlc-done",
+    namespace: str = "u-test",
+    phase: str = "succeeded",
+    rank_exits: list = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)